kafka-handler/src/resources/SimpleRecord.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "type" : "record",
+  "name" : "SimpleRecord",
+  "namespace" : "org.apache.hadoop.hive.kafka",
+  "fields" : [ {
```

Comment:
I'd be interested in seeing more complex tests, for example, mapping Avro `doc` fields to the Hive `comment` fields

Reply:
I'm not familiar with `doc` fields and `comment` fields. Can u elaborate a little more or provide some documentation.

Reply:
It's not really relevant for this PR, but as part of the DDL, we have `col_comment`

```
CREATE [TEMPORARY] [EXTERNAL] TABLE [IF NOT EXISTS] [db_name.]table_name    -- (Note: TEMPORARY available in Hive 0.14.0 and later)
  [(col_name data_type [COMMENT col_comment], ... [constraint_specification])]
 ```

And if you look at the Avro spec, you'll see `doc` can be a field added, much like JavaDoc comments for classes and fields

Reply:
Agreed, but this seems tangential to the issue at hand. It could be committed without additional tests on the schema.